pkg/token/kubeconfig.go

```diff
@@ -78,9 +78,11 @@ func CreateKubeletBootstrapConfig(clusterConfig *config.ClusterConfig, k0sVars c
 	if role == "worker" {
 		data.User = "kubelet-bootstrap"
 		data.JoinURL = clusterConfig.Spec.API.APIAddressURL()
-	} else {
+	} else if role == "controller" {
```

Comment:
Use the added consts for this and above worker ase

Reply:
it is a different package I will move the constants at the top of this file